.circleci/config.yml

```diff
@@ -33,6 +37,16 @@ jobs:
       - restore_cache:
           key: identity-dashboard-{{ checksum "Gemfile.lock" }}
 
+      - run:
```

Comment:
These changes are necessary because the container we use for CircleCI appears to be pinned to an older version of Node.js not supported by the version of `eslint-config-airbnb` upgraded through these changes:

```
error eslint-config-airbnb@19.0.4: The engine "node" is incompatible with this module. Expected version "^10.12.0 || ^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.16.0"
```

The changes here will install the latest v14 version (v14.19.3 as of writing).

This is also the same as what we have in the IdP, and matches the version of Node in the build environments.

- https://github.com/18F/identity-idp/blob/main/.nvmrc
- https://github.com/18F/identity-idp/blob/222d56af/.circleci/config.yml#L42-L54